### PR TITLE
feat: Add profanity filter toggle

### PR DESCRIPTION
**Pull request by pilanop**
324282876-0ac9bf20-4348-463d-ace0-dc9a56b55853

### CODE REVIEW

**Comment by RobiMez** [comment]
🫡 Reviewing

File: src/routes/b/[room]/+page.svelte [hunk 1]
```diff
@@ -208,7 +212,7 @@
 	{#if sent}
 		<span class="text-xl font-light">✨ Sent your message </span>
 	{:else if profanityWarning}
-		<span class="text-error"> ⚠️ Message has not been sent because it contains profanity </span>
+		<span style="color: #c95050;">⚠️ Message has not been sent because it has profanity words</span>
```
Comment (RobiMez) [inline src/routes/b/[room]/+page.svelte:215]: ... why ?

Comment (RobiMez) [reply src/routes/b/[room]/+page.svelte:215]: contains profanity is more like . .. grammatically correct

Comment (pilanop) [reply src/routes/b/[room]/+page.svelte:215]: U're right I was planning changing it to "the host doesn't allow profanity words"

Comment (pilanop) [reply src/routes/b/[room]/+page.svelte:215]: ⚠️ Message has not been sent because the room host doesn't allow profanity words

Comment (RobiMez) [reply src/routes/b/[room]/+page.svelte:215]: Yeah that makes a lot more sense , thanks lol

**Comment by RobiMez** [comment]
using local storage to handle the state of profanityFilterEnabled makes it so the filter only works on your browser and not the other peoples ... gotta persist it in the database.

**Comment by RobiMez** [comment]
lemme just quickly do that for you

**Comment by RobiMez** [comment]
we probably should move the profanity check to the server but this will do for now

**Pull request merged**